backend/connector-integration/src/utils.rs

```diff
@@ -1,8 +1,54 @@
-use domain_types::connector_types::PaymentsAuthorizeData;
+use domain_types::connector_PaymentsAuthorizeData;
 use hyperswitch_interfaces::errors;
 
 type Error = error_stack::Report<errors::ConnectorError>;
 
+pub mod xml_utils;
+pub use xml_utils::preprocess_xml_response_bytes;
+
+use std::{
+    collections::{HashMap, HashSet},
+    str::FromStr,
+    sync::LazyLock,
+};
+
+use api_models::{
+    enums::{CanadaStatesAbbreviation, UsStatesAbbreviation},
+    payments,
+};
+use base64::Engine;
+use common_utils::{
+    date_time,
+    errors::{ParsingError, ReportSwitchExt},
+    ext_traits::StringExt,
+    id_type,
+    pii::{self, Email, IpAddress},
+    types::{AmountConvertor, MinorUnit},
+};
+use diesel_models::{enums, types::OrderDetailsWithAmount};
+use error_stack::{report, ResultExt};
+use hyperswitch_domain_models::{
```

Comment:
We should remove this dependency also and we should not directly use this PaymentAttempt